docs/spec/parallel_dask.rst

```diff
@@ -0,0 +1,72 @@
+.. _parallel_dask:
+
+============================
+Parallel Execution with Dask
+============================
+
+You should understand :ref:`Kartothek's Querying Process <querying_process>`.
+
+Queries
+-------
+
+Kartothek executes queries with a task for every parquet file to read. These
+files are filtered by the predicate's restrictions on partitions and indexed
+columns.
+
+The example dataset in the following pictures is partitioned on column ``A``.
+The bottom half shows the existing parquet files, the upper half shows the
+querying process with the (possibly) created tasks. The data is filtered for
+``A=2 AND B="b"``. ``A=2/label1.parquet`` and ``A=2/label2.parquet`` are the
+only files to be touched because of the restriction on the partitioned column
+``A``.
+
+.. image:: /images/kartothek_read_dispatch.png
+  :width: 600
+  :alt: Kartothek dispatches tasks by parquet file
+
+
+For every file its range of values (stated in the parquet footer) is matched
+against the query.
+
+Given ``RowGroup Chunksize >= 2``, for ``A=2/label1.parquet`` the range for
+column ``B`` is ``Min: "a"; Max: "a"`` and for ``A=2/label2.parquet`` it is
+``Min: "a"; Max: "b"``. So only ``A=2/label2.parquet`` contains relevant rows.
```

Comment:
"After reading the Parquet statistics for ``A=2/label1.parquet``, we can rule out the existence of a row with `B="b"` in this file (as min/max is ...). Thus, we do not read any data from that file.
We only read data from the file A=2/label2.parquet."